include/ghex/unstructured/pattern.hpp

```diff
@@ -0,0 +1,346 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+#ifndef INCLUDED_GHEX_UNSTRUCTURED_PATTERN_HPP
+#define INCLUDED_GHEX_UNSTRUCTURED_PATTERN_HPP
+
+#include <vector>
+#include <map>
+#include <cassert>
+#include <numeric>
+#include <cstring>
+
```

Comment:
```suggestion
#include <iosfwd>

```
?

Reply:
Not sure.. Why would I need it here?

Reply:
you have a std::ostream reference in line 151, with this you ensure the declaration of that is available, and you would not need to include iostream if you do not need to use. Not only, this would make the inclusion of iostream not be forced to happen before this file

Reply:
Yep, true. I pushed it to the new PR.